### PR TITLE
Add Oracle Linux 8 in Binary Tarball Job 

### PR DESCRIPTION
**Pull request by kaushikpuneet07**

### CODE REVIEW

**Comment by kaushikpuneet07** [comment]
Successfull Job Link : https://ps80.cd.percona.com/view/QA/job/test-ps-binary-tarball/236/console

File: binary-tarball-tests/ps/test-ps-binary-tarball.groovy [hunk 1]
```diff
@@ -133,7 +133,7 @@ void run_test() {
     else
       sudo apt install -y git wget lsb-release
     fi
-    git clone https://github.com/Percona-QA/package-testing.git --branch master --depth 1
+    git clone https://github.com/kaushikpuneet07/package-testing.git --branch os-update --depth 1
```
[the remark this answers is not in the document]
Comment (kaushikpuneet07) [reply binary-tarball-tests/ps/test-ps-binary-tarball.groovy]: Fixed

File: binary-tarball-tests/ps/run.sh [hunk 2]
```diff
@@ -4,6 +4,7 @@ export PATH=${HOME}/.local/bin:${PATH}
 echo "Installing dependencies..."
 if [ -f /etc/redhat-release ]; then
   sudo yum install -y libaio numactl openssl
+  sudo yum install -y tar
```
Comment (hrvojem) [inline binary-tarball-tests/ps/run.sh:7]: tar can be added to the line above

**Review by hrvojem — CHANGES_REQUESTED** [review]
minor changes requested

File: binary-tarball-tests/ps/run.sh (hunk 2 above)
Comment (kaushikpuneet07) [reply binary-tarball-tests/ps/run.sh]: Fixed

File: binary-tarball-tests/ps/run.sh [hunk 3]
```diff
@@ -24,6 +25,8 @@ else
   else
     sudo apt install -y python3 python3-pip
   fi
+  sudo apt-get update -y
+  sudo apt-get install -y libldap-2.4-2
   sudo apt install -y libaio1 libnuma1
```
[the remark this answers is not in the document]
Comment (kaushikpuneet07) [reply binary-tarball-tests/ps/run.sh]: Merged

**Review by hrvojem — APPROVED** [review]
LGTM

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**